charter.html

```diff
@@ -285,8 +371,8 @@ <h2 id=process>Process</h2>
 <a href=https://www.w3.org/Consortium/cepc/>W3C Code of Ethics and
 Professional Conduct</a>.
 
-<p>Contributions to Deliverables can only be made by Community Group
-Participants who have agreed to the
+<p>Contributions to <a href=#work-items>Work Items</a> can only be made
```

Comment:
This is probably also true for Proposals. You'll want a clear IP trail for everything in the group.

Reply:
good catch; fixed in b641830.